addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.scss

```diff
@@ -11,9 +11,16 @@
         background-color: white;
         height: 18px;
         width: 18px;
+        @media screen  and (max-width: 480px) {
+            height: 13px;
+            width: 13px;
+        }
     }
 }
 
 .o_avatar_card {
-    width: 380px;
+    max-width: 380px;
+    @media screen  and (max-width: 480px) {
+        max-width: 90vw;
+    }
```

Comment:
Ok for now, I see that `90vw` is wisely chosen to make sure popover direction stays bottom, or the popover is necessarily out-of-bound. Popover reposition wants to necessarily preserve anchor so it just attempts best N/E/S/W even if this results in cut content.

Again the value `90vw` works by assuming the triggers of avatar card popover have a least an available distance to left/right border that can put the card vertically from anchor on at least 90vw.

@aab-odoo is there plan to improve popover positioning further, or is this preferably improved after a future bigger improvement, e.g. using browser native Popover API under-the-hood of popover components?